ush/forecast_postdet.sh

```diff
@@ -581,37 +616,59 @@ MOM6_out()
 CICE_postdet()
 {
 	echo "SUB ${FUNCNAME[0]}: CICE after run type determination"
+
+        year=$(echo $CDATE|cut -c 1-4)
+        #BL2018
+        stepsperhr=$((3600/$ICETIM))
+        #BL2018
+        #nhours=$(${NHOUR} ${CDATE} ${SYEAR}010100)
+        nhours=$($NHOUR $CDATE ${year}010100)
+        istep0=$((nhours*stepsperhr))
+        steps=$((nhours*stepsperhr))
+        npt=$((FHMAX*$stepsperhr))      # Need this in order for dump_last to work
+
+        histfreq_n=${histfreq_n:-6}
+        restart_interval=${restart_interval:-1296000}    # restart write interval in seconds, default 15 days
+        dumpfreq_n=$restart_interval                     # restart write interval in seconds
+
+        #BL2018
+        #dumpfreq='d'
+        #dumpfreq='s'
+        if [ -d $ROTDIR/../NEXT_IC ]; then
```

Comment:
This looks like this is adding things in from GODAS workflow.  

I thought the "NEXT_IC" directories in GODAS were going to be changed to point to more typical storage locations with COMROT?

Reply:
It may not be the final design for the godas side. But it won't affect the current features/coupled-crow by any means either. So I suggest keeping it at this time.